examples/regressions.ml

```diff
@@ -32,6 +32,7 @@ let compare name (old_l : (int * int) list) (new_l : Sedlex_ppx.Sedlex_cset.t) =
     code_points
 
 let test new_l (name, old_l) =
+  let old_l = Sedlex_utils.Cset.to_list old_l in
```

Comment:
I think it would be better to manipulate cset directly. See #159

Reply:
It still fails with `master` code (should be renamed `main` too). This call is just a pass-through. Looks like defining the type as `private` requires it. Again, not sure that this really matters..

Reply:
```ocaml
type t = private (int * int) list

let to_list l = l
```